src/authorizations/components/redesigned/TokenRow.tsx

```diff
@@ -136,28 +137,37 @@ class TokensRow extends PureComponent<Props> {
     this.props.deleteAuthorization(id, description)
   }
 
-  private handleClone = () => {
+  private handleClone = async () => {
     const {description} = this.props.auth
-
     const allTokenDescriptions = Object.values(this.props.authorizations).map(
       auth => auth.description
     )
 
-    this.props.createAuthorization({
-      ...this.props.auth,
-      description: incrementCloneName(allTokenDescriptions, description),
-    })
-    this.props.showOverlay('access-token', null, () => dismissOverlay())
+    try {
+      await this.props.createAuthorization({
+        ...this.props.auth,
+        description: incrementCloneName(allTokenDescriptions, description),
+      })
+      event('token.clone.success', {id: this.props.auth.id, name: description})
```

Comment:
This event being here makes sense, because clone and create use the same thunk so we want to differentiate between the two somehow.